common/defs/bitcoin/blockstamp.json

```diff
@@ -0,0 +1,44 @@
+{
+  "coin_name": "BlockStamp",
+  "coin_shortcut": "BST",
+  "coin_label": "BlockStamp",
+  "website": "https://blockstamp.info",
+  "github": "https://github.com/BlockStamp/bst",
+  "maintainer": "The BlockStamp Developers <support@blockstamp.info>",
+  "curve_name": "secp256k1",
+  "address_type": 26,
+  "address_type_p2sh": 5,
+  "maxfee_kb": 2000000,
+  "minfee_kb": 1000,
+  "signed_message_header": "BST Signed Message:\n",
+  "hash_genesis_block": "8000000049a2e26b0185be50b4b8ed58b707c8893762959f0b1673641cae1828",
+  "xprv_magic": 76066276,
+  "xpub_magic": 76067358,
+  "xpub_magic_segwit_p2sh": 77429938,
+  "xpub_magic_segwit_native": 78792518,
+  "bech32_prefix": "bst",
+  "cashaddr_prefix": null,
+  "slip44": 254,
+  "segwit": true,
+  "decred": false,
+  "fork_id": null,
+  "force_bip143": false,
+  "bip115": false,
+  "default_fee_b": {
+    "Low": 10,
+    "Economy": 70,
+    "Normal": 140,
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 60,
+  "uri_prefix": "blockstamp",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [
+    "https://explorer.blockstamp.info"
```

Comment:
This is not a valid bitcore backend; remove this entry

Reply:
Removed.